### ISSUE TITLE
Enhancement: Proper week 6 resolution

### ISSUE THREAD

**Issue by Bigboibigdum** [FunkinCrew/Funkin#2534]
## What is your suggestion, and why should it be implemented?

As a pixel artist week 6 is pretty cool
except
MIXELS NOOO WHY HERE

Week 6 uses pixel art graphics for its assets but the actual game isn't at a resolution where an asset pixel is the same size as an actual game pixel so when a sprite moves it creates mixels. I don't know how easily you can change the resolution but if there was some limit for week 6 that would look alot better imo

(also could you maybe resize the hold arrow lines to be a bit thicker please i scream in fear whenever i see them)

**Comment by ninjamuffin99** [comment]
I thiiiink for the most part most of the pixel art is scaled up 6x, but you are right that when the camera pans the pixels move in a non pixel perfect way! it's def something I've poked at before, but haven't done any full viewing comparisons. but its officially in our task backlog now :)

**Comment by EliteMasterEric** [comment]
We reviewed a couple PRs in the past which made changes to the camera in Week 6, but they only made things look stuttery.

Can you clarify on what exactly the issue is?

**Comment by Bigboibigdum** [comment]
I think you could do like a background subpixel thing where the actual camera movement is normal but visually it only moves when it's a proper pixel away from where it currently is? Here's a filler sentence because ending it like that just doesn't feel right im sorry!

**Comment by Punkinator7** [comment]
[Codename Engine](https://github.com/FNF-CNE-Devs/CodenameEngine) does this! Here's [a video](https://youtu.be/IgEpfPoPBSQ) I recorded because I could not find footage of it anywhere else.

**Comment by Bigboibigdum** [comment]
Yeah that's pretty much what I'm thinking of. It is a bit stuttery, but honestly i prefer it over 

**the evil small square next to big square combo**
